src/main/java/org/javawebstack/orm/query/Query.java

```diff
@@ -296,59 +319,10 @@ public Query(Repo<T> repo, Class<T> model) {
         return this;
     }
 
-    public QueryString getQueryString() {
```

Comment:
This confuses me. Removing this should break the tests for the newly merged query test tooling, but it seems the test are running.

Reply:
I already merged the tests from master into the maintenance branch and did the necessary fixes. The replacement for this is `query.getRepo().getConnection().builder().buildQuery(query, false)` which now uses the query builder for the dialect of the connection.

Reply:
Makes sense. Great!